media/js/base/core-datalayer-init.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global initDownloadLinks, updateDownloadTextForOldFx, initMobileDownloadLinks,
```

Comment:
You can remove these global eslint definitions as they are not part of this file.